### ISSUE TITLE
polygon edge style

### ISSUE THREAD

**Issue by furvius** [tomchadwin/qgis2web#238]
If "no border line" option of a polygon is selected, when you activate that layer it makes disappear all the visible layers on the map.

```
var styleCache_ZTOBCittconsolidata={}
var style_ZTOBCittconsolidata = function(feature, resolution){
    var value = ""
    var size = 0;
    var style = [ new ol.style.Style({
        stroke: new ol.style.Stroke({color: "rgba(0,0,0,0.596078)", lineDash: [no], lineCap: 'butt', lineJoin: 'miter', width: 0}) , fill: new ol.style.Fill({color: "rgba(255,127,0,0.596078)"})
    })];

```
That's why:

> Uncaught ReferenceError: no is not defined 
> style_ZTOBCittconsolidata	@	ZTOBCittconsolidata_style.js:8

**Comment by tomchadwin** [comment]
Confirmed in OL3 output - works in Leaflet. Working on a fix.

**Comment by tomchadwin** [comment]
Fixed in master - please test. However, I think stroke styles (dashed, dotted, etc.) are now broken in OL3. Will close this and open another ticket for that.

**Comment by tomchadwin** [comment]
Released as 1.5.1 and pushed to QGIS plugins repo.
